mne/decoding/transformer.py

```diff
@@ -715,3 +719,149 @@ def transform(self, X):
+        - l_freq is None, h_freq is not None: high-pass filter
+
+    See ``mne.filter.filter_data``.
+
+    Parameters
+    ----------
+    l_freq : float | None
+        Low cut-off frequency in Hz. If None the data are only low-passed.
+    h_freq : float | None
+        High cut-off frequency in Hz. If None the data are only
+        high-passed.
+        channels will be filtered.
+    sfreq : float, defaults to 1.0
+        Sampling frequency in Hz.
+    filter_length : str | int, defaults to 'auto'
+        Length of the FIR filter to use (if applicable):
+            * int: specified length in samples.
+            * 'auto' (default in 0.14): the filter length is chosen based
+              on the size of the transition regions (7 times the reciprocal
+              of the shortest transition band).
+            * str: (default in 0.13 is "10s") a human-readable time in
+              units of "s" or "ms" (e.g., "10s" or "5500ms") will be
+              converted to that number of samples if ``phase="zero"``, or
+              the shortest power-of-two length at least that duration for
+              ``phase="zero-double"``.
+    l_trans_bandwidth : float | str, defaults to 'auto'
+        Width of the transition band at the low cut-off frequency in Hz
+        (high pass or cutoff 1 in bandpass). Can be "auto"
+        (default in 0.14) to use a multiple of ``l_freq``::
+            min(max(l_freq * 0.25, 2), l_freq)
+        Only used for ``method='fir'``.
+    h_trans_bandwidth : float | str, defaults to 'auto'
+        Width of the transition band at the high cut-off frequency in Hz
+        (low pass or cutoff 2 in bandpass). Can be "auto"
+        (default in 0.14) to use a multiple of ``h_freq``::
+            min(max(h_freq * 0.25, 2.), info['sfreq'] / 2. - h_freq)
+        Only used for ``method='fir'``.
+    n_jobs : int | str, defaults to 1
+        Number of jobs to run in parallel. Can be 'cuda' if scikits.cuda
+        is installed properly, CUDA is initialized, and method='fft'.
+    method : str, defaults to 'fir'
+        'fir' will use overlap-add FIR filtering, 'iir' will use IIR
+        forward-backward filtering (via filtfilt).
+    iir_params : dict | None, defaults to None
+        Dictionary of parameters to use for IIR filtering.
+        See mne.filter.construct_iir_filter for details. If iir_params
+        is None and method="iir", 4th order Butterworth will be used.
+    verbose : bool, str, int, or None, defaults to None
+        If not None, override default verbose level (see mne.verbose).
+        Defaults to self.verbose.
+
+    See Also
+    --------
+    FilterEstimator
+    Vectorizer
+    mne.filter.band_pass_filter
+    mne.filter.band_stop_filter
+    mne.filter.low_pass_filter
+    mne.filter.high_pass_filter
+    """
+    def __init__(self, l_freq=None, h_freq=None, sfreq=1.0,
+                 filter_length='auto', l_trans_bandwidth='auto',
```

Comment:
Setting the values of `l_trans_bandwidth` and `high_trans_bandwidth` to 'auto' is making the python3 test in travis fail. However it should not occur.
@kingjr @Eric89GXL @mmagnuski @agramfort